### DISCUSSION TITLE
Encoding error on displayed stack trace while using FXGL

### DISCUSSION THREAD

**Discussion by Nassima04** [AlmasB/FXGL#1115] [Q&A]
Hello, I'm building an application with JDK11, I'm using Eclipse, JavaFx and the library FXGL. 

I have a problem while executing my program : 
When I have an error, it's displayed with a different language that I don't understand (seems to be some unicode characters) as you can see on the capture bellow.

I want to ask you if you know how solve this encoding problem and make my errors appear in English.
I ask you because I don't have this issue without using the FXGL library even if perhaps it's not direct linked to FXGL.
<img width="652" alt="ERROR REPORTER" src="https://user-images.githubusercontent.com/92041032/140729694-8e1e8d8f-8a67-446f-9952-1b40dc6258cf.png">

Thanks a lot for your help !

**Comment by AlmasB** [comment]
Hi, this is really odd -- first time I'm seeing this. The text to be displayed is indeed in English in the source:

https://github.com/AlmasB/FXGL/blob/e02535d8df6f83071f951e7085bcab22e918a440/fxgl/src/main/kotlin/com/almasb/fxgl/app/scene/ErrorSubScene.kt#L71-L75

Try building the application from the command line and see if the issue reappears. It might be something to do with Eclipse / Maven combination, for example [see this link](https://stackoverflow.com/questions/25404145/register-font-is-corrupting-ttf-file) -- not the same issue but similar.

**Comment by Nassima04** [comment]
Hi @AlmasB ,

Thanks a lot for your response ! It helps me even if building the application from the command line didn't fix the problem.

I changed the Maven version used on Eclipse (I downloaded a 3.8.3 instead of my current 3.6.3) and using the maven command : mvn javas:run (in Eclipse) the language issue doesn't appear anymore.

But if I run with Run As > Java application I still have the issue and the same using the command line. Really strange :/ 

Anyway, with this fix I can work ! Thanks
